example/pky_serial_sites.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO `redcap_entity_test_site` (`created`,`updated`,`site_long_name`,`site_short_name`,`site_address`,`site_appointment_duration`,`open_time`,`close_time`,`closed_days`,`horizon_days`,`testing_type`,`project_id`,`start_date`,`custom_daily_schedule`,`use_custom_daily_schedule`) VALUES (1587089441,1587502827,'PK Yonge Tent 5','5','1080 SW 11th St, Gainesville, FL 32601','30','12:30','13:00','6','40','swabandserum',1,1587960000,'{\r\n    \"Monday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Tuesday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Wednesday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Thursday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Friday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Saturday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Sunday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    }\r\n}','1');
+INSERT INTO `redcap_entity_test_site` (`created`,`updated`,`site_long_name`,`site_short_name`,`site_address`,`site_appointment_duration`,`open_time`,`close_time`,`closed_days`,`horizon_days`,`testing_type`,`project_id`,`start_date`,`custom_daily_schedule`,`use_custom_daily_schedule`) VALUES (1587089441,1587502827,'PK Yonge Tent 6','6','1080 SW 11th St, Gainesville, FL 32601','30','12:30','13:00','6','40','swabandserum',1,1587960000,'{\r\n    \"Monday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Tuesday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Wednesday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Thursday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Friday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Saturday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Sunday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    }\r\n}','1');
```

Comment:
```suggestion
INSERT INTO `redcap_entity_test_site` (`created`,`updated`,`site_long_name`,`site_short_name`,`site_address`,`site_appointment_duration`,`open_time`,`close_time`,`closed_days`,`horizon_days`,`testing_type`,`project_id`,`start_date`,`custom_daily_schedule`,`use_custom_daily_schedule`) VALUES (UNIX_TIMESTAMP(),UNIX_TIMESTAMP(),'PK Yonge Tent 1','1','1080 SW 11th St, Gainesville, FL 32601','30','12:30','13:00','6','40','swabandserum',1,UNIX_TIMESTAMP(CURDATE()),'{\r\n    \"Monday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Tuesday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Wednesday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Thursday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Friday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Saturday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Sunday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    }\r\n}','1');
INSERT INTO `redcap_entity_test_site` (`created`,`updated`,`site_long_name`,`site_short_name`,`site_address`,`site_appointment_duration`,`open_time`,`close_time`,`closed_days`,`horizon_days`,`testing_type`,`project_id`,`start_date`,`custom_daily_schedule`,`use_custom_daily_schedule`) VALUES (UNIX_TIMESTAMP(),UNIX_TIMESTAMP(),'PK Yonge Tent 2','2','1080 SW 11th St, Gainesville, FL 32601','30','12:30','13:00','6','40','swabandserum',1,UNIX_TIMESTAMP(CURDATE()),'{\r\n    \"Monday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Tuesday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Wednesday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Thursday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Friday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Saturday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Sunday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    }\r\n}','1');
INSERT INTO `redcap_entity_test_site` (`created`,`updated`,`site_long_name`,`site_short_name`,`site_address`,`site_appointment_duration`,`open_time`,`close_time`,`closed_days`,`horizon_days`,`testing_type`,`project_id`,`start_date`,`custom_daily_schedule`,`use_custom_daily_schedule`) VALUES (UNIX_TIMESTAMP(),UNIX_TIMESTAMP(),'PK Yonge Tent 3','3','1080 SW 11th St, Gainesville, FL 32601','30','12:30','13:00','6','40','swabandserum',1,UNIX_TIMESTAMP(CURDATE()),'{\r\n    \"Monday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Tuesday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Wednesday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Thursday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Friday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Saturday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Sunday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    }\r\n}','1');
INSERT INTO `redcap_entity_test_site` (`created`,`updated`,`site_long_name`,`site_short_name`,`site_address`,`site_appointment_duration`,`open_time`,`close_time`,`closed_days`,`horizon_days`,`testing_type`,`project_id`,`start_date`,`custom_daily_schedule`,`use_custom_daily_schedule`) VALUES (UNIX_TIMESTAMP(),UNIX_TIMESTAMP(),'PK Yonge Tent 4','4','1080 SW 11th St, Gainesville, FL 32601','30','12:30','13:00','6','40','swabandserum',1,UNIX_TIMESTAMP(CURDATE()),'{\r\n    \"Monday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Tuesday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Wednesday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Thursday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Friday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Saturday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Sunday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    }\r\n}','1');
INSERT INTO `redcap_entity_test_site` (`created`,`updated`,`site_long_name`,`site_short_name`,`site_address`,`site_appointment_duration`,`open_time`,`close_time`,`closed_days`,`horizon_days`,`testing_type`,`project_id`,`start_date`,`custom_daily_schedule`,`use_custom_daily_schedule`) VALUES (UNIX_TIMESTAMP(),UNIX_TIMESTAMP(),'PK Yonge Tent 5','5','1080 SW 11th St, Gainesville, FL 32601','30','12:30','13:00','6','40','swabandserum',1,UNIX_TIMESTAMP(CURDATE()),'{\r\n    \"Monday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Tuesday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Wednesday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Thursday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Friday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Saturday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Sunday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    }\r\n}','1');
INSERT INTO `redcap_entity_test_site` (`created`,`updated`,`site_long_name`,`site_short_name`,`site_address`,`site_appointment_duration`,`open_time`,`close_time`,`closed_days`,`horizon_days`,`testing_type`,`project_id`,`start_date`,`custom_daily_schedule`,`use_custom_daily_schedule`) VALUES (UNIX_TIMESTAMP(),UNIX_TIMESTAMP(),'PK Yonge Tent 6','6','1080 SW 11th St, Gainesville, FL 32601','30','12:30','13:00','6','40','swabandserum',1,UNIX_TIMESTAMP(CURDATE()),'{\r\n    \"Monday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Tuesday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Wednesday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Thursday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Friday\": {\r\n        \"open\": \"12:30\",\r\n        \"close\": \"13:00\"\r\n    },\r\n    \"Saturday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    },\r\n    \"Sunday\": {\r\n        \"open\": \"\",\r\n        \"close\": \"\"\r\n    }\r\n}','1');
```
Makes `created` and `updated` timestamps set to when the query was run; sets the `start_date` to the day the query was run.

Reply:
Thank you for doing that which I was too busy to even think about. I had just said "nope, not today".  I hadn't imagined it would be so easy. :-)

Reply:
You just needed to remove the `NOT` from `TODAY()`

Reply:
@pbchase this suggestion doesn't appear to have been committed, just the conversation was marked as resolved - error on your part?

Reply:
weird.  github had a brain fart. I already used their tool and made commit 2489d4d below. I'll try my luck withe commit suggestion button.